Makefile

```diff
@@ -19,12 +22,18 @@ clean:
 	@echo "--> Cleaning up ./build"
 	@rm -rf build/*
 
-## install: Build and install the celestia-node binary into the GOBIN directory.
-install:
+## install: Build and install the celestia-node binary into the $PREFIX (/usr/local/ by default) directory.
+install: build
 	@echo "--> Installing Celestia"
-	@go install -ldflags ${LDFLAGS}  ./cmd/celestia
+	@sudo install -v ./build/* -t ${PREFIX}/bin/
```

Comment:
This makes the calling shell save the sudo credentials. I.e. the next time I do anything that requires `sudo` it will pass silently. Not sure this is better behavior than explicitly require `make install` to be called with `sudo`.